crates/ln-dlc-node/src/disk.rs

```diff
@@ -22,3 +23,16 @@ pub(crate) fn read_scorer(
     }
     ProbabilisticScorer::new(params, graph, logger)
 }
+
+pub(crate) fn read_network(
+    path: &Path,
+    genesis_hash: BlockHash,
+    logger: Arc<TracingLogger>,
+) -> NetworkGraph {
+    if let Ok(file) = File::open(path) {
+        if let Ok(graph) = NetworkGraph::read(&mut BufReader::new(file), logger.clone()) {
+            return graph;
+        }
```

Comment:
Should we log the error at least?

Reply:
yes, I think the network graph is also updated through the gossip protocol.